Content.Server/Speech/EntitySystems/SkeletonAccentSystem.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Text.RegularExpressions;
+using Content.Server.Speech.Components;
+using Robust.Shared.Random;
+
+namespace Content.Server.Speech.EntitySystems;
+
+public sealed class SkeletonAccentSystem : EntitySystem
+{
+    [Dependency] private readonly IRobustRandom _random = default!;
+
+    private static readonly Dictionary<string, string> DirectReplacements = new()
+    {
+        { "fuck you", "I've got a BONE to pick with you" },
+        { "fuck", "rattle rattle" },
+        { "shit", "rattle rattle" },
+        { "definitely", "make no bones about it" },
+        { "absolutely", "make no bones about it" },
+        { "lonely", "bonely"}
+    };
```

Comment:
This doesn't actually conform to any particular order, did you want a list of tuples instead?

Reply:
Would tuples help? I genuinely don't know enough about them to tell. Prior to my latest commit, the string `"fuck you"` would be replaced with `"rattle rattle you"` instead of `"I've got a BONE to pick with you"`, but now in testing they both work as intended.

I double checked, and the only other accent system that uses direct replacements like this is `MobsterAccentSystem` and that uses a simple `Dictionary<string, string>` like this does.

Reply:
dict seems fine here? if you need defined order a list of tuples might be better

Reply:
In that case I'll resolve this one. But I'll keep tuples in mind for more complicated systems where defined order would be needed!